sdk/digitaltwins/digitaltwins/samples/Readme.md

```diff
@@ -0,0 +1,282 @@
+  newTwin
+);
+```
+
+### Get a digital twin
+
+Getting a digital twin is extremely easy.
+
+```Javascript Snippet:dt_digitaltwins_lifecycle
+const digitalTwinId = "myTwin";
+const twin = await serviceClient.getDigitalTwin(digitalTwinId);
+console.log(`DigitalTwin's etag: ${twin.eTag}`);
+console.log(`DigitalTwin: ${twin.body}`);
+```
+
+### Query digital twins
+
+Query the Azure Digital Twins instance for digital twins using the [Azure Digital Twins Query Store lanaguage](https://review.docs.microsoft.com/en-us/azure/digital-twins-v2/concepts-query-language?branch=pr-en-us-114648). Query calls support paging. Here's an example of how to query for digital twins and how to iterate over the results.
+
+```Javascript Snippet:dt_digitaltwins_query
+const query = "SELECT * FROM digitaltwins";
+const queryResult = serviceClient.queryTwins(query);
+for await (const item of queryResult) {
+  console.log(`DigitalTwin: ${item}`);
+}
+```
+
+### Delete digital twins
+
+Delete a digital twin simply by providing Id of a digital twin as below.
+
+```Javascript Snippet:dt_digitaltwins_lifecycle
+const digitalTwinId = "myTwin";
+await serviceClient.deleteDigitalTwin(digitalTwinId);
+```
+
+## Get and update digital twin components
+
+### Update digital twin components
+
+To update a component or in other words to replace, remove and/or add a component property or subproperty within Digital Twin, you would need Id of a digital twin, component name and application/json-patch+json operations to be performed on the specified digital twin's component. Here is the sample code on how to do it.
+
+```Javascript Snippet:dt_component_lifecycle
+// Update Component1 by replacing the property ComponentProp1 value
+const digitalTwinId = "myTwin";
+const componentPath = "Component1";
+const jsonPatch = {
+  ComponentProp1: "value2"
+};
+await serviceClient.updateComponent(digitalTwinId, componentPath, jsonPatch);
+```
+
+### Get digital twin components
+
+Get a component by providing name of a component and Id of digital twin to which it belongs.
+
+```Javascript Snippet:DigitalTwinsSampleGetComponent
+const digitalTwinId = "myTwin";
+const componentPath = "Component1";
+const component = await serviceClient.getComponent(digitalTwinId, componentPath);
+console.log(`Updated component: ${component}`);
+```
+
+## Create and list digital twin relationships
+
+### Create digital twin relationships
+
+`upsertRelationship` creates a relationship on a digital twin provided with Id of a digital twin, name of relationship such as "contains", Id of an relationship such as "FloorContainsRoom" and an application/json relationship to be created. Must contain property with key "\$targetId" to specify the target of the relationship. Sample payloads for relationships can be found [here](https://github.com/Azure/azure-sdk-for-net-pr/blob/feature/IoT-ADT/sdk/iot/Azure.Iot.DigitalTwins/samples/DigitalTwinServiceClientSample/DTDL/Relationships/HospitalRelationships.json "RelationshipExamples").
```

Comment:
update link to relationship example

Reply:
Fixed

---
In reply to: [455393702](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r455393702) [](ancestors = 455393702)